pkg/apis/v1alpha2/labels.go

```diff
@@ -119,8 +118,6 @@ var (
 		'c': LabelSKUConfidential,
 		'i': LabelSKUIsolatedSize,
 	}
-
-	NodeClaimLinkedAnnotationKey = v1alpha5.MachineLinkedAnnotationKey // still using the one from v1alpha5
```

Comment:
I did see anywhere we reference this, but is it needed elsewhere that we should keep this around for some reason?

Would need to hardcode the value now though.

Reply:
No, this value is useless now without Link Controller @rakechill  can correct me if im wrong